src/interface/meshblock_data.hpp

```diff
@@ -52,11 +52,16 @@ class MeshBlockData {
   /// Constructor
   MeshBlockData<T>() = default;
 
+  /// Copies variables from src, optionally only copying names given and/or variables
+  /// matching any of the flags given. If shallow_copy is true, no copies of variables
+  /// will be allocated regardless whether they are flagged as OneCopy or not
+  void CopyFrom(const MeshBlockData<T> &src, bool shallow_copy,
+                const std::vector<std::string> &names = {},
+                const std::vector<MetadataFlag> &flags = {});
```

Comment:
They both act as filters, so if both are specified, it will only copy variables that show up in `names` (either full label or sparse base name) and also have metadata that has at least one flag in `flags` set.

Reply:
I attempted to explain it succinctly as `
Copies variables from src, optionally only copying names given and/or variables matching any of the flags given.`. Do you have a suggestion for how to make this clearer?